api.js

```diff
@@ -1,11 +1,36 @@
 const axios = require('axios');
 
+const salesforceInstance = axios.create({
+  baseURL: `${process.env.SERVICE_CLOUD_URI}/api/v1/`,
+  headers: {
+    'Content-Type': 'application/json'
+  }
+});
+
 module.exports = {
   
   createTechOps: async function(payload) {
     return axios.post(process.env.API_URL + '/techops', payload, requestHeader())
   },
-  
+
+  createSalesforceCase: async function(payload, authResponse) {
+    return salesforceInstance.post(
+      "cases",
+      payload,
+      requestServiceCloudAppHeader(authResponse)
+    );
+  },
+
+  authenticateSalesforce: async function() {
+    const credentialsPayload = JSON.stringify({ username: process.env.SERVICE_CLOUD_USER, password: process.env.SERVICE_CLOUD_PASSWORD });
+
+    return salesforceInstance.post(
+      "authenticate",
+      credentials_payload,
```

Comment:
Faltou renomear aqui também 😁

```suggestion
      credentialsPayload,
```

Reply:
Ahhh Obrigada 🙏 !! Vi depois que fiz o commit 🤦‍♀️